Makefile

```diff
@@ -17,6 +17,7 @@ ALL_SRC := $(shell find . -name '*.go' \
 							-not -path './bin/*' \
 							-not -path './build/*' \
 							-not -path './tools/linters/*' \
+							-not -path './tools/release/*' \
```

Comment:
let's keep your tool src code for lint scan. :)

Reply:
Ok, I will remove this exclusion.